tests/snapshots/tests.contrib.httpx.test_httpx.test_configure_global_service_name_env.snap

```diff
@@ -0,0 +1,19 @@
+[[{"name" "http.request"
```

Comment:
What's the purpose of these snap files? I'm assuming it's a sample trace but why don't we just use a json or txt file

Reply:
We have a "test agent" which provides the same API as our trace agent (e.g. `POST /v0.4/traces` endpoint to receive traces), but it will do some assertions on the traces it receives to make sure the traces we are producing are valid and meet the required shared requirements.

What it also does is store fixture files of the traces produced from "snapshot tests". The first time you run the test it'll save this `.snap` file to disk with this representation of what the trace(s) it received during that test were. Then every consecutive time you run the test it uses this `.snap` file to assert the traces it receives matches what is in these files.

So these snap files say to expect 1 span with a given name/service/meta/metrics/etc. We can use this instead of writing a ton of individual asserts on the resulting spans.